beacon_node/client/src/notifier.rs

```diff
@@ -417,231 +410,139 @@ async fn bellatrix_readiness_logging<T: BeaconChainTypes>(
 }
 
 /// Provides some helpful logging to users to indicate if their node is ready for Capella
-async fn capella_readiness_logging<T: BeaconChainTypes>(
+async fn post_capella_readiness_logging<T: BeaconChainTypes>(
```

Comment:
For clarity I think this should be:
```suggestion
async fn post_bellatrix_readiness_logging<T: BeaconChainTypes>(
```

Reply:
Fixed with https://github.com/sigp/lighthouse/pull/6797/commits/af49dc8c0fbd65abb10474c426f52b7df3945dea